.pre-commit-config.yaml

```diff
@@ -17,7 +17,7 @@ repos:
     rev: v0.0.286
     hooks:
       - id: ruff
-        args: [--fix, --exit-non-zero-on-fix]
+        args: [--fix, --exit-non-zero-on-fix, --ignore=PD002]
```

Comment:
Please modify ruff.toml instead

Reply:
Fixed in 53310f27f7e1942da0ad9c7efe9d74b37ec2f680